src/FileOrganizerSettingTab.ts

```diff
@@ -137,6 +137,17 @@ export class FileOrganizerSettingTab extends PluginSettingTab {
             await this.plugin.saveSettings();
           })
       );
+    new Setting(containerEl)
+      .setName("Apply AI Template Formatting")
+      .setDesc("Automatically format your documents using your AI templates (when applicable) during file processing.")
+      .addToggle((toggle) =>
+        toggle
+          .setValue(this.plugin.settings.aiTemplateFormatting)
+          .onChange(async (value) => {
```

Comment:
prevent user from activating this if user does not have early access features

aka

1. check does user have early this.settings.enableearly access
2. no -> notif only for early access toggle.setDisabled(true) this.settings.aitemplateformat = false -> save
3. yes -> this.settings.aitempalte = true -> save

Reply:
see example of "enableEarlyaccess" setting to see how to manipulate toggle state